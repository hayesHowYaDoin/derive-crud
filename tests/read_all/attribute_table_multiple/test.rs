use derive_crud::ReadAll;

#[derive(ReadAll)]
#[crud_table("test_table")]
#[crud_table("test_table_2")]
struct Test {
    #[crud_id]
    id: i64,
    name: String,
}

#[tokio::main]
async fn main() {
    let _results = Test::read_all().await;
}
