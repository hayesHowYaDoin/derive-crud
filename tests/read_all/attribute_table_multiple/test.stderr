error: Only one `#[crud_table]` attribute is allowed
 --> tests/read_all/attribute_table_multiple/test.rs:5:3
  |
5 | #[crud_table("test_table_2")]
  |   ^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0599]: no function or associated item named `read_all` found for struct `Test` in the current scope
  --> tests/read_all/attribute_table_multiple/test.rs:17:26
   |
6  | struct Test {
   | ----------- function or associated item `read_all` not found for this struct
...
17 |     let _results = Test::read_all(&pool).await;
   |                          ^^^^^^^^ function or associated item not found in `Test`
